### DISCUSSION TITLE
VLC

### DISCUSSION THREAD

**Discussion by joaochris** [sircharlo/meeting-media-manager#278] [Q&A] [answered]
_Create playlists for use with VLC_
How does this option work?

**Comment by riemsesy** [comment]
Doesn't JWMMF create an XSPF playlist file?

**Reply by joaochris** [reply]
> Doesn't JWMMF create an XSPF playlist file?

Now I found out that yes hahaha

**Comment by Chipperchap** [comment, marked as answer]
It is very cool. When enabled, and after "Get Media", go to the meeting folder, in there is a VLC  .xspf file. Copy/paste, drag/drop into the VLC Playlist box, click on the Playlist and everything media related for the meeting is in the Playlist automatically. Great for those who want to use VLC as their media player as per organization suggestion.

**Reply by joaochris** [reply]
> É muito legal. Quando habilitado, e após "Obter Mídia", vá para a pasta da reunião, onde há um arquivo VLC .xspf. Copie/cole, arraste/solte na caixa VLC Playlist, clique na Playlist e todas as mídias relacionadas à reunião estarão na Playlist automaticamente. Ótimo para quem deseja usar o VLC como media player conforme sugestão da organização.

Wow, I hadn't seen that! Thanks for explaining.
